resources/public/api/conf/1.0/application.yaml

```diff
@@ -37,7 +37,7 @@ components:
   securitySchemes:
     User-Restricted:
       type: oauth2
-      description: HMRC supports OAuth 2.0 for authenticating [User-restricted](https://developer.qa.tax.service.gov.uk/api-documentation/docs/authorisation/user-restricted-endpoints) API requests
+      description: HMRC supports OAuth 2.0 for authenticating [User-restricted](https://developer.tax.service.hmrc.gov.uk/api-documentation/docs/authorisation/user-restricted-endpoints) API requests
```

Comment:
Hi Abigail, The URL is not valid. it should be https://developer.service.hmrc.gov.uk/api-documentation/docs/authorisation/user-restricted-endpoints